README.md

```diff
@@ -1 +1,13 @@
-https://en.wikipedia.org/wiki/Nim
+[Learn about Nim][nim]
+
+# development
+
+### setup
+`mkvirtualenv nim --python=python3.6`
```

Comment:
Should we explain how to install `python3.6`, `mkvirtualenv` and `pip`?

Reply:
Maybe links?